neqo-transport/src/send_stream.rs

```diff
@@ -1369,6 +1380,27 @@ impl SendStream {
     pub(crate) fn state(&mut self) -> &mut SendStreamState {
         &mut self.state
     }
+
+    pub(crate) fn emit_writable_event(&mut self, previous_limit: usize, current_limit: usize) {
```

Comment:
Here limit can be (1) the connection flow control limit, (2) the stream flow control limit, or (3) the tx buffer limit. See call sites.